dbt_subprojects/solana/models/solana_utils/solana_utils_token_accounts_timing.sql

```diff
@@ -0,0 +1,97 @@
+{{
+  config(
+        schema = 'solana_utils',
+        alias = 'token_accounts_timed',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['address', 'token_balance_owner', 'token_mint_address', 'valid_from', 'valid_to'],
```

Comment:
We'll need to remove the `valid_to` column from the unique key to be able to override older records with an updated `valid_to` value when a new address combo is activated.